### ISSUE TITLE
Understanding 4.1.3 Status Messages (Level AA)

### ISSUE THREAD

**Issue by nrhsinclair** [w3c/wcag-eo#5]
Post suggested revisions here.

**Comment by a11ycob** [comment]
My comments for the this SC. Overall I found this one was fairly straight-forward with little changes required.
[status-messages.docx](https://github.com/w3c/wcag-eo/files/2756618/status-messages.docx)

**Comment by cochfarf** [comment]
•	The link to Changes of context should be added the first time it is used near the link to status message and not later in the page. I went looking for the definition before I saw it was 3 paragraphs late in the copy
•	What exactly does "surfaced by the screen reader" mean? Does it need to be defined here, or linked to a definition?

**Comment by bakkenb** [comment]
- (not part of the understanding content, but still an issue on this page) - Three of the four words that are glossed (underlined for definition), the links lead to the WCAG 2.1 home page and not to the term itself.
- Fourth paragraph down, "Status Message Examples" link goes nowhere.
- Agree with Lewis about surfaced by assistive technology. May say detected and voiced by assistive technology. This is used twice on the page.
- I think the subheading of "Scope" should be added before the third paragraph.
- Fifth paragraph down, "Examples of Changes That Are Not Status Messages" link goes nowhere.

Most explanation in this one is straight forward and easy to understand, even for non-developer readers.

**Comment by nrhsinclair** [comment]
I agree this document is mainly clear and in this need of revision. I agree "surfaced by" is not clear enough, perhaps "exposed to Assistive Technologies or screen readers" would be better.

I also think "exploited by assistive technologies" should be changed to "utilized".

**Comment by nrhsinclair** [comment]
I think the opening intent statement could be stronger: instead of "The intent of this Success Criterion is _to make users aware of important changes_ in content that are not given focus, and to do so in a way that doesn't unnecessarily interrupt their work."

The intent of this Success Criterion is to _ensure that users are made aware_ of important changes in content that are not given focus, and to do so in a way that doesn't unnecessarily interrupt their work.
